src/winmain.cpp

```diff
@@ -492,24 +509,46 @@ bool runInstaller(const string& app2runPath, const string& binWindowsClassName,
 
 		if (h)
 		{
-			int installAnswer = ::MessageBoxA(NULL, closeMsg.c_str(), closeMsgTitle.c_str(), MB_YESNO);
+			if (false == IsUpdateInBackground)

```

Comment:
Ugly style, I prefer to not use it.

Reply:
> Ugly style

What do you mean? `if (false == IsUpdateInBackground)` should be `if (!IsUpdateInBackground)` or something else?